docs/CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# the following will be requested for
+# review when someone opens a pull request.
+*       @LukeShirnia @dsgnr @LukeHandle
+
+# In this example, @doctocat owns any files in the build/logs
```

Comment:
This comment refers to an example which is no longer present

Reply:
Fixed. Thanks